apps/frontend/src/main.tsx

```diff
@@ -1,19 +1,38 @@
 import { StrictMode } from 'react';
+import i18next from 'i18next';
+import resourcesToBackend from 'i18next-resources-to-backend';
 import * as ReactDOM from 'react-dom';
 
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { ReactQueryDevtools } from 'react-query/devtools';
 
 import App from './app/app';
 
-const queryClient = new QueryClient();
+i18next
```

Comment:
We might use react-i18next module, see docs
https://react.i18next.com/guides/the-drawbacks-of-other-i18n-solutions

It has over 1mln weekly downloads on npm

Reply:
fixed mordo